packages/mux-audio/src/CustomAudioElement.js

```diff
@@ -67,19 +67,15 @@ class CustomAudioElement extends HTMLElement {
 
     this.attachShadow({ mode: 'open' });
     this.shadowRoot.appendChild(template.content.cloneNode(true));
-    const nativeEl = (this.nativeEl = this.shadowRoot.querySelector('audio'));
 
-    // Initialize all the attribute properties
-    Array.prototype.forEach.call(this.attributes, (attrNode) => {
-      this.attributeChangedCallback(attrNode.name, null, attrNode.value);
-    });
+    const nativeEl = (this.nativeEl = this.shadowRoot.querySelector('audio'));
 
     // Neither Chrome or Firefox support setting the muted attribute
     // after using document.createElement.
     // One way to get around this would be to build the native tag as a string.
     // But just fixing it manually for now.
     // Apparently this may also be an issue with <input checked> for buttons
-    if (nativeEl.defaultMuted) {
+    if (this.hasAttribute('muted')) {
```

Comment:
this is needed because we aren't running the above attributes loop which would set `nativeEl.defaultMuted` to the correct value. 

`attributeChangedCallback` is always called automatically when the custom element is upgraded and there are some attributes set so the loop is really not required.